### DISCUSSION TITLE
架构即政治——电子前沿基金会介绍 - 开源之道

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#17] [Announcements]
# 架构即政治——电子前沿基金会介绍 - 开源之道

https://opensourceway.community/posts/foundation_introduce/introduction_of_electronic_frontier_foundation/

**Comment by ZoomQuiet** [comment]
#是也乎,(￣▽￣)
推荐另外一本相关的神书: 杜鹃蛋：电脑间谍案曝光录 (The Cuckoo's Egg)
 https://book.douban.com/subject/6870140/
出版时, 根本不知道应该如何翻译 IT 技术图书, 很多术语 比如 Unix 是全部用 尤尼克斯 来指代的;
记录了互联网早期 黑客攻防战斗的真实过程,
电传打字机才是 嗅探结果输出设置...

从中, 可以大致知道, IEEE/EFF 们成立的时代, 涉及互联网的管理机构是什么构成, 如何运行的,
以及冷战背景中, 世界真实的心理反应是如何的...

其实,开源之史的创作, 最困难的, 应该是找到一种共情形式,
来合理让当前以及未来的读者可以舒畅的感受到过去时代中,
对于未知事物时,
当时最合适的反应为什么是那样;

以及, 这种民间发起, 慢慢结合大企业/政府, 最终变成全球通行行业标准的过程中,
什么力量在起核心动力?

俺的偏见是 大学为主的学术共同体, 
自古形成的学派力量...

而这种力量的聚集, 又是校友制度的积累;
传说, 国外大学办校目的是:
+ 令大学是校友一生中, 最快乐的时光
+ 这样, 在毕业后, 根本无法忘怀, 一直和学校有联系, 一但有点儿成就, 就主动自豪捐赠母校
当然, 中国解放后断绝了大学独立性, 变成行政单位,
大学时代变成多数人一生最后悔的时光..然后, 就没有然后了;

这也导致开源在中国水土不服, 毕竟多数成年人, 开蒙之后根本没机会体验自主结社/协作创新,
从幼儿园开始就一直是集体生活, 有严明的纪律和要求,
怎么可能和雅皮文化为基调的开源文化融合?

那么, 问题来了, 这种协会的发生/发展历史, 从中应该摄取什么经验来反哺或是触发我们本土的 FLOSS?
